### DISCUSSION TITLE
ERPGulf Announcing the launch of the Saudi Zatca-2 E-Invoicing app on FrappeCloud

### DISCUSSION THREAD

**Discussion by fkalpana** [ERPGulf/saudi-phase-2#6] [Announcements]
We're thrilled to announce the launch of the **Zatca-2 E-Invoicing app on FrappeCloud**! In this release, we've removed all Java and SDK dependencies, ensuring full compatibility with FrappeCloud

In this release, we've removed all Java and SDK dependencies, ensuring full compatibility with FrappeCloud's robust security features. We've also refactored the code for enhanced performance and reliability. plus, it's 100% open-source.

A huge shoutout to the Zatca account managers for their invaluable advice, our amazing clients for their cooperation during the development and testing phases, and a special thanks FrappeCloud team

A heartfelt thank you to our incredible development team, for their dedication and hard work in making this launch a success.
Addressed two major concerns from our Saudi clients regarding the move to FrappeCloud:

1. **Data Sovereignty: ** Frappe has launched a new data center in Jeddah. (Data sovereignty law requires Saudi companies to host financial data inside Saudi Arabian geographical boundaries.)

2. **Missing E-Invoicing App: ** We've published the Zatca-2 app.

Also, we are adding other Govt integrations like Visa processes, Insurance, Medical, and Payment gateways for Saudi ERPNext users. Keep watching this space.

With these solutions in place, this is the perfect time to transition from hosting companies outside Saudi Arabia to FrappeCloud (Jeddah) to ensure legal compliance in both data sovereignty and e-invoicing, as well as for better latency and speed. Our team can help you to migrate.

As we continue to enhance the Zatca-2 E-Invoicing app, we’d love to hear from you! If you have any suggestions for bug fixes or feature additions, please lets know. Your input is invaluable in helping us create the best experience possible.

Join us in celebrating this milestone and experience the new and improved Zatca-2 E-Invoicing app!

**Comment by fkalpana** [comment]
https://cloud.erpgulf.com/blog/news/announcing-the-launch-of-the-saudi-zatca-2-e-invoicing-app-on-frappecloud

**Comment by omarjabr** [comment]
Thank you guys for your great work, I have a small question. Which one should I install zatca_erpgulf or Saudi-E-Invoicing-Phase-2-2024?

**Reply by fkalpana** [reply]
zatca_erpgulf

**Reply by omarjabr** [reply]
great thank you so much!
